test/fixtures/allowed-paths.txt

```diff
@@ -1022,7 +988,6 @@
 /hlx_e5dcc1db3f1f11bebf7fc35de9f6f199560076bc.jpeg
 /hlx_26909954f61aa359412f1e8f024678c2850116de.jpeg
 /hlx_ee505e436bf92041978210dc3082fc708e4837f7.jpg
-/office/%E6%97%A5%E6%9C%AC.html
```

Comment:
What do you have against Korean?

Reply:
we don't support unicode.... nobody uses it anyways.